src/dirs.proj

```diff
@@ -15,6 +15,10 @@
     <Project Include="shims\ApiCompat.proj" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(DirectoryToBuild)' != ''">
+    <Project Include="$(DirectoryToBuild)/**/*.csproj" />
```

Comment:
I'm definitely focused on making a shortcut here as opposed to a filtering mechanism. I'm not sure I want to plumb filtering through these projects, at least not at this point. I do think it is worth considering the ref/src/packaging/tests ordering but I was trying to make this a little more general so you could build any directory and so baking in that assumption might cause other scenarios to break. I think I will keep things as they are for now and let folks play around with it and adjust based on feedback.  Another option I might consider is to drop a dirs.proj in to the root of each of the library directories to control things like ordering and other project types.